build/go.mk

```diff
@@ -85,6 +85,10 @@ promote-to-stable:
 		--key tel2/$(GOHOSTOS)/$(GOHOSTARCH)/stable.txt \
 		--body $(BUILDDIR)/stable.txt
 
+.PHONY: publish-homebrew-formula
+publish-homebrew-formula:
+	packaging/homebrew-package.sh $(patsubst v%,%,$(TELEPRESENCE_VERSION))
```

Comment:
Can we put this in `build/` instead of making a new directory?

Reply:
Hmm, on second thought `packaging/` might be a good addition as we grow new things.

Reply:
Yeah I'm not opposed to putting it in the `build/` dir, just didn't know if it would get to be _too_ much, especially once we add the linux + potentially windows stuff one day. But I'm also fine moving it to the `build/` dir to start and then we can break it out if it gets to be confusing.